build/pack.cake

```diff
@@ -80,6 +87,11 @@ Task("Test")
 
         Parallel.Invoke(options, actions.ToArray());
     }
+
+    var workDir = "./src/GitVersionVsixTask";
+    var npmSettings = new NpmRunScriptSettings { WorkingDirectory = workDir, LogLevel = NpmLogLevel.Silent, ScriptName = "test" };
+    npmSettings.Arguments.Add($"--reporter-options mochaFile={MakeAbsolute(new FilePath($"{testResultsPath}vsix.results.xml"))}");
```

Comment:
Having a lot of statements inside a string interpolation makes it hard to read. What do you think about this solution?

```suggestion
    npmSettings.Arguments.Add($"--reporter-options mochaFile={absoluteTestResultsPath}");
```

Reply:
Looks good to me, I'll adjust, thanks